test/errors/test_FPDF_errors.py

```diff
@@ -107,6 +109,14 @@ def test_repeated_calls_to_output(tmp_path):
     assert_pdf_equal(pdf, HERE / "repeated_calls_to_output.pdf", tmp_path)
 
 
+def test_unsupported_image_filter_error():
+    img = Image.open(HERE / "flowers.png")
+    image_filter = "N/A"
+    with pytest.raises(FPDFException) as error:
+        _to_data(img, image_filter)
+    assert str(error.value) == f'Unsupported image filter: "{image_filter}"'
+
+    
```

Comment:
Seems like Pylint does not like some trailing whitespaces here:
```
test/errors/test_FPDF_errors.py:119:0: C0303: Trailing whitespace (trailing-whitespace)
```

Reply:
I added the pre-commit hook with pylint and black for next times